examples/visualization/styles/deck-styles.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" type="text/css" href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css">
+
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    #map {
+      width: 100vw;
+      height: 100vh;
+    }
+  </style>
+</head>
+
+<body>
+  <section id="map"></section>
+
+  <!-- Map libraries-->
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.js"></script>
+  <script src="https://unpkg.com/deck.gl@8.1.0/dist.min.js"></script>
+
+  <!-- CARTO Deck.gl's Tile Layer -->
+  <script src="/packages/toolkit/dist/umd/index.min.js"></script>
+
+  <script>
+    async function initialize() {
+      carto.setDefaultCredentials({ username: 'public' });
+
+      const deckMap = carto.viz.createMap({
+        view: {
+          longitude: -116.846099,
+          latitude: 50.2672297,
+          zoom: 2.5
+        }
+      });
+
+      const styles = {
+        getFillColor: [130,109,186, 229],
+        getLineWidth: 1,
+        getLineColor: [255,255,255,127]  
+      };
+
+      const countriesLayer = new carto.viz.Layer('ne_50m_admin_1_states_1', styles);
```

Comment:
How are we handling this styles that are just a plan object? I've just seen that our helpers now return a `Style` class instance.

Reply:
Yes, the styles could a Style class instance or a plain object.